packages/e2e/test/web-extension/dapp-sdk/features/setExpiredValidityIntervalTx.ts

```diff
@@ -0,0 +1,36 @@
+import { EMPTY, catchError, take, tap } from 'rxjs';
+import { inspectAndSignTx } from '../utils';
+
+import { Cardano } from '@cardano-sdk/core';
+import type { ObservableWallet } from '@cardano-sdk/wallet';
+
+export const setExpiredValidityIntervalTx = async ({
+  logger,
+  connectedWallet
+}: {
+  logger: typeof console;
+  connectedWallet: ObservableWallet;
+}) => {
+  const sendInfoElement = document.querySelector('#info-send')!;
+
+  connectedWallet.tip$
+    .pipe(
+      take(1),
+      tap(async (tip) => {
+        const validityInterval: Cardano.ValidityInterval = { invalidHereafter: Cardano.Slot(tip.blockNo) };
+
+        const builder = connectedWallet.createTxBuilder();
+
+        const builtTx = builder.addOutput(await builder.buildOutput().handle('rhys').coin(10_000_000n).build());
+
+        const expiredValidityIntervalTx = builtTx.setValidityInterval(validityInterval).build();
+
+        inspectAndSignTx({ builtTx: expiredValidityIntervalTx, connectedWallet, textElement: sendInfoElement });
+      }),
+      catchError((error) => {
+        logger.error('Error in fetching tip of chain', error);
```

Comment:
That might be misleading
```suggestion
        logger.error('Failed to build tx', error);
```

Reply:
it's addressed here: 6169d4881e48a53c5c5065cdc8a2d1b804d7902b